ports/nrf/Makefile

```diff
@@ -74,12 +74,14 @@ CFLAGS += $(CFLAGS_MCU_$(MCU_SERIES))
 CFLAGS += $(INC) -Wall -Werror -ansi -std=gnu99 -nostdlib $(COPT) $(NRF_DEFINES) $(CFLAGS_MOD)
 CFLAGS += -fno-strict-aliasing
 CFLAGS += -fstack-usage
+CFLAGS += -fdata-sections -ffunction-sections
```

Comment:
This duplicates the same setting for cortex-m4 targets in line 68. As this is made common for cortex-m0/m4, we should probably remove the old declaration of this.

Reply:
Fixed.
That line for the M4 will only have increased code size: `-ffunction-sections -fdata-sections` without `--gc-sections` produces larger code than without those switches.

Reply:
Oh! i did not know that, better good we got rid of it then! :D